ci-operator/config/redhat-developer/rhdh/redhat-developer-rhdh-main.yaml

```diff
@@ -1,17 +1,7 @@
 build_root:
-  image_stream_tag:
-    name: release
-    namespace: openshift
-    tag: golang-1.20
-  use_build_cache: true
-images:
-- context_dir: .ibm/images
-  dockerfile_path: Dockerfile
-  to: showcase-e2e-runner
-promotion:
-  to:
-  - name: rhdh
-    namespace: ci
+  project_image:
```

Comment:
We can speed up the jobs by mirroring the images - see: https://docs.ci.openshift.org/docs/how-tos/external-images/

To add an image to the mirror you need to open a PR like that one: https://github.com/openshift/release/pull/61185 and ask for #forum-ocp-testplatform Slack channel using the button at the bottom. You can then use the image like this: https://github.com/openshift/release/pull/61168/files#diff-b04b3d172a4ac92bead3647b6e171309d8eb3215aca41c42e9b4a8f003be2371R8-R11

I'm not sure how quickly a new version of the `rhdh-e2e-runner` would propagate to the OpenShift CI mirror, but with the mirror, we won't have to wait to pull the image every time.